GW2EIEvtcParser/EncounterLogic/Strikes/SotO/TempleOfFebe.cs

```diff
@@ -92,27 +100,30 @@ protected override List<int> GetTargetsIDs()
                 (int)TrashID.PermanentEmbodimentOfMalice,
                 (int)TrashID.PermanentEmbodimentOfRage,
                 (int)TrashID.PermanentEmbodimentOfRegret,
+                (int)TrashID.MaliciousShadow,
```

Comment:
Does the normal mode shadow need to be in target?

Reply:
I just wanted it consistent with cm before people report us as bug "malicious shadow shows in cm but not nm" :D